src/Daemon/Daemon.cpp

```diff
@@ -70,6 +72,8 @@ namespace usbguard
     { "apply-policy", Daemon::DevicePolicyMethod::ApplyPolicy }
   };
 
+  static int pid_fd = -1;
```

Comment:
Is it necessary to have this variable defined outside the Deamon class scope? If it needs to be a global variable it can still be defined inside the Daemon class as a static variable.